vendor/github.com/keybase/client/go/libkb/identify_outcome.go

```diff
@@ -225,12 +225,12 @@ func (i IdentifyOutcome) GetErrorAndWarnings(strict bool) (warnings Warnings, er
 
 	if ntf := i.NumTrackFailures(); ntf > 0 {
 		probs = append(probs,
-			fmt.Sprintf("%d track component%s failed",
+			fmt.Sprintf("%d tracked proof%s failed",
```

Comment:
Asked @malgorithms, he agreed with you John. https://github.com/keybase/client/pull/5731 (This code doesn't run in KBFS, but I'll bump this PR if that one goes in first.)